test/check-ostree

```diff
@@ -245,6 +245,8 @@ class OstreeRestartCase(testlib.MachineCase):
         m.upload(["files/publickey.asc"], "/root/")
         m.execute("ostree remote gpg-import local -k /root/publickey.asc")
 
+        testlib.sit()
```

Comment:
This obviously does not belong in the PR :)

Reply:
Yup, already resolved locally, I also have some commit message fixups. I'm mostly trying to hunt down [this flake](https://cockpit-logs.us-east-1.linodeobjects.com/pull-560-b8ee88f8-20240327-065534-fedora-coreos/log.html), which at first sight *feels* unrelated, but also is new.

Reply:
I added a commit which fixes it for me locally.